### PR TITLE
Protocol relative urls

### PR DESCRIPTION
**Pull request by craigdallimore**
Hello

These tweaks enable the browser to determine whether to use `https` or `http` for the dynamically loaded scripts based on the protocol of the page.

This means they will happily load from a locally running dev server (http) or a github.io [example](https://craigdallimore.github.io/repl/) page. (https).

Additional information: https://www.paulirish.com/2010/the-protocol-relative-url/

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -131,12 +129,12 @@
...
             scripts: [
               {
-                src      : 'http://wzrd.in/standalone/sanctuary@latest',
+                src      : '//wzrd.in/standalone/sanctuary@latest',
                 global   : 'sanctuary',
                 exposeAs : 'S'
               },
               {
-                src    : 'http://wzrd.in/standalone/ramda-fantasy@latest',
+                src    : '//wzrd.in/standalone/ramda-fantasy@latest',
```
Comment (davidchambers) [inline index.html]: Does wzrd.in support HTTPS?

Comment (craigdallimore) [inline index.html]: Yes - example: [https://wzrd.in/standalone/sanctuary@latest](https://wzrd.in/standalone/sanctuary@latest)

You can also observe it using your network panel on [this page](https://craigdallimore.github.io/repl). (Requires a click to start the repl).

**Comment by davidchambers** [comment]
:deciduous_tree:

**Comment by buzzdecafe** [comment]
🐪

**Pull request merged**